src/snowcli/cli/snowpark_shared.py

```diff
@@ -81,31 +77,35 @@ def snowpark_package(
             )
             if do_download:
                 log.info("Installing non-Anaconda packages...")
-                should_pack, second_chance_results = utils.install_packages(
+                should_continue, second_chance_results = utils.install_packages(
                     "requirements.other.txt",
```

Comment:
Let's use this occassion to introduce constant for this file name, I think we use it at least twice

Reply:
Done